bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.ts

```diff
@@ -314,11 +320,33 @@ export class KtbEvaluationDetailsComponent implements OnInit, OnDestroy {
   }
 
   _chartSeriesClicked(event) {
-    this._selectedEvaluationData = event.point.evaluationData;
+    this.selectEvaluationData(event.point.evaluationData);
   }
 
   _heatmapTileClicked(event) {
-    this._selectedEvaluationData = this._heatmapSeries[0].data[event.point.x]['evaluation'];
+    this.selectEvaluationData(this._heatmapSeries[0].data[event.point.x]['evaluation']);
+  }
+
+  selectEvaluationData(evaluation) {
+    this._selectedEvaluationData = evaluation;
+    this.highlightHeatmap();
+  }
+
+  highlightHeatmap() {
+    let highlightIndex = this._heatmapOptions.xAxis[0].categories.indexOf(moment(this._selectedEvaluationData.time).format());
+    let plotBands = [];
+    if(highlightIndex >= 0)
+      plotBands.push({
+        borderWidth: 2,
+        borderColor: '#e6be00',
+        color: 'rgba(230,190,0,.4)',
+        from: highlightIndex-0.5,
+        to: highlightIndex+0.5,
+        zIndex: 20
+      });
+    this._heatmapOptions.xAxis[0].plotBands = plotBands;
+    this.heatmapChart._update();
```

Comment:
I'm getting an error here if I switch between evaluation events too quickly:

`ERROR TypeError: this.heatmapChart is undefined`

this results in the component not being rendered correctly:
![Screenshot from 2020-09-23 17-05-52](https://user-images.githubusercontent.com/56065213/94031376-115b5300-fdbf-11ea-81e1-ba3eed70ccde.png)

Reply:
Yeah, definitely I need only to update the heatmap, if it is already created. Will fix that.